chromadb/segment/impl/metadata/grpc_segment.py

```diff
@@ -81,6 +81,12 @@ def get_metadata(
     ) -> Sequence[MetadataEmbeddingRecord]:
         """Query for embedding metadata."""
 
+        if limit is not None and limit < 0:
+            raise ValueError("Limit cannot be negative")
```

Comment:
I prefer a ChromaError with a specific error code

Reply:
I've added a new `OutOfBoundError` for this scenario